packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -128,11 +128,11 @@ const startRecurrence: MutationResolvers['startRecurrence'] = async (
     analytics.recurrenceStarted(viewerId, meetingSeries)
   } else {
     const newMeetingSeries = await startNewMeetingSeries(viewerId, teamId, meetingId)
-    dataLoader.get('newMeetings').clear(meetingId)
-
     analytics.recurrenceStarted(viewerId, newMeetingSeries)
   }
 
+  dataLoader.get('newMeetings').clear(meetingId)
```

Comment:
This fixes the issue when the meeting object is stale when restarting the recurrence